mt-metadata/.htaccess

```diff
@@ -3,5 +3,5 @@ Options +FollowSymLinks
 
 RewriteEngine On
 
-# Add redirect for testing via demo RVA server and using placeholder domain: http://example.com/nci_second_example_for_jo/
-RewriteRule ^(.*)$ https://demo.vocabs.ardc.edu.au/repository/api/lda/nci/replacement-metadata-standards-for-magnetotelluric/auscope-auslamp-profile/resource?uri=http://example.com/nci_second_example_for_jo/$1 [B,L,NE,QSA,R=307]
+# Add redirect for testing via demo RVA server and using new W3ID domain: http://w3id/mt-metadata/profiles/auscope-auslamp/v0.1.2/
+RewriteRule ^(.*)$ https://demo.vocabs.ardc.edu.au/repository/api/lda/nci/replacement-metadata-standards-for-magnetotelluric/auscope-auslamp-profile/resource?uri=http://w3id/$1 [B,L,NE,QSA,R=307]
```

Comment:
- You should use `https://w3id.org/...`
- Did you mean to have `mt-metadata/` in that `uri=` param?